core/nginx/conf/nginx.conf

```diff
@@ -29,10 +29,10 @@ http {
     # Main HTTP server
     server {
       # Variables for proxifying
-      set $admin admin;
-      set $antispam antispam:11334;
-      set $webmail webmail;
-      set $webdav webdav:5232;
+        set $admin {{ HOST_ADMIN }};
```

Comment:
You need the same change [here](https://github.com/benjaminmisell/Mailu/blob/20785a808a1c07c05a0de641e0b440db51c7eff5/core/nginx/conf/nginx.conf#L138).